app/models/like.rb

```diff
@@ -2,7 +2,7 @@
 
 class Like < ApplicationRecord
   belongs_to :post
-  belongs_to :user
+  belongs_to :author
```

Comment:
- [ ] There is no model named `Author`, so we need to specify the model being referred to here. Based on the feedback from the previous reviewer we can add the `class_name` attribute.

  ```
  belongs_to :author, class_name: 'User'
  ```